# The function pow() is used without including the header file, which leads to compilation errors.
# By including math.h this issue is fixed.
#
# Tom Strempel, UFZ
--- src/mira/ads.C.orig	2020-06-22 14:05:17.506971632 +0200
+++ src/mira/ads.C	2020-06-22 14:06:06.113071330 +0200
@@ -39,6 +39,8 @@
 #include "errorhandling/errorhandling.H"
 #include "util/dptools.H"
 
+#include <math.h>
+
 using namespace std;
 
 #define CEBUG(bla)
